python/python/glide/async_commands/core.py

```diff
@@ -3043,6 +3045,87 @@ async def xack(
             await self._execute_command(RequestType.XAck, [key, group_name] + ids),
         )
 
+    async def xpending(
+        self,
+        key: str,
+        group_name: str,
+    ) -> List[Union[int, str, List[List[str]], None]]:
+        """
+        Returns stream message summary information for pending messages for the given consumer group.
+
+        See https://valkey.io/commands/xpending for more details.
+
+        Args:
+            key (str): The key of the stream.
+            group_name (str): The consumer group name.
+
+        Returns:
+            List[Union[int, str, List[List[str]], None]]: A list that includes the summary of pending messages, with the
```

Comment:
In which condition it returns `None`?

Reply:
If there are no pending messages for the group, there's a test [here](https://github.com/aws/glide-for-redis/pull/1704/files#diff-1c3a6923c1bf3dd19a1c98c6523f881876a156a8316c3b3f5dc6801a405382ceR5767) that shows this

Reply:
Can you mention this in the doc or show an example?